README.md

```diff
@@ -11,6 +11,16 @@ Welcome to Braintree's Drop-in SDK for iOS!
 
 **The Braintree iOS Drop-in SDK permits a deployment target of iOS 12.0 or higher.** It requires Xcode 15.0+ and Swift 5.9+.
 
+## 📣 Announcements
+
+**The iOS Drop-In SDK is currently inactive and will be unsupported by November 2025**. As of December 2024, the iOS Drop-In SDK is inactive. No additional features will be added with the exception of security updates. As an alternative, we recommend using the ![Braintree SDK](https://github.com/braintree/braintree_ios) for unbranded checkout.
```

Comment:
Nit / take it or leave it - instead of "recommend" (which suggests they maybe look to another provider) do we want to say something like

"Instead, implement a custom UI using braintree_ios"?

Reply:
Good point, will add that to be more specific

Reply:
Should I add that we are no longer addressing bugs? or leave it at "no additional features will be added"?

I changed the last sentence to say "Instead, implement a custom UI using the ![Braintree SDK (braintree_ios)](https://github.com/braintree/braintree_ios)." Open to feedback about the wording